Makefile

```diff
@@ -32,8 +32,8 @@ help:
 .env:
 	@if [ ! -f .env ]; \
 	then \
-	echo "Copying env.dist to .env..."; \
-	cp docker/config/env.dist .env; \
+	echo "Copying .env.example to .env"; \
+	cp .env.example .env; \
```

Comment:
I don't see why this has to be so complicated. 
The rule isn't executed if `.env` exists...

Example:
https://github.com/mozilla-services/telescope/blob/f4739c65d7309ef39bdebd3c198e618ade89fc36/Makefile#L44-L45

Reply:
Good catch. I was mostly just find/replacing, but good opportunity to fix this